src/EnergyPlus/Furnaces.hh

```diff
@@ -142,48 +142,41 @@ namespace Furnaces {
-        bool MaxHeatAirVolFlowEMSOverrideOn;            // if true, EMS is calling to override autosize flow during heating
-        Real64 MaxHeatAirVolFlowEMSOverrideValue;       // EMS value for override of flow during heating operation [m3/s]
-        Real64 MaxNoCoolHeatAirVolFlow;                 // supply air volumetric flow rate when no cooling or heating [m3/s]
-        bool MaxNoCoolHeatAirVolFlowEMSOverrideOn;      // if true, EMS is calling to override autosize no heatcool rate
-        Real64 MaxNoCoolHeatAirVolFlowEMSOverrideValue; // EMS value for override of flow during no heat cool [m3/s]
-        Real64 MaxCoolAirMassFlow;                      // supply air mass flow rate during cooling operation [kg/s]
-        Real64 MaxHeatAirMassFlow;                      // supply air mass flow rate during heating operation [kg/s]
-        Real64 MaxNoCoolHeatAirMassFlow;                // supply air mass flow rate when no cooling or heating [kg/s]
-        Real64 MaxHeatCoilFluidFlow;                    // water or steam mass flow rate for heating coil [kg/s]
-        Real64 MaxSuppCoilFluidFlow;                    // water or steam mass flow rate for supplemental heating coil [kg/s]
-        Real64 ControlZoneMassFlowFrac;                 // Fraction of furnace flow to control zone
-        Real64 DesignMaxOutletTemp;                     // Maximum supply air temperature from furnace heater [C]
-        Real64 MdotFurnace;                             // Mass flow rate through furnace [kg/s]
-        Real64 FanPartLoadRatio;                        // Part load ratio of furnace fan (mdot actual/mdot design)
-        Real64 CompPartLoadRatio;                       // Part load ratio of furnace compressor (load / steady-state output)
-        Real64 WSHPRuntimeFrac;                         // Runtime fraction of water source heat pump
-        Real64 CoolPartLoadRatio;                       // Cooling part load ratio
-        Real64 HeatPartLoadRatio;                       // Heating part load ratio
-        Real64 MinOATCompressorCooling;                 // Minimum outdoor operating temperature for heat pump compressor
-        Real64 MinOATCompressorHeating;                 // Minimum outdoor operating temperature for heat pump compressor
-        Real64 MaxOATSuppHeat;                          // Maximum outdoor dry-bulb temperature for
-        int CondenserNodeNum;                           // Node number of outdoor condenser/compressor
-        Real64 MaxONOFFCyclesperHour;                   // Maximum ON/OFF Cycling Rate [cycles/hr]
-        Real64 HPTimeConstant;                          // Heat Pump Time Constant [s]
-        Real64 OnCyclePowerFraction;                    // Fraction of on-cycle power use [~]
-        // supplemental heating coil operation
-        Real64 FanDelayTime; // Fan delay time, time delay for the HP's fan to
-        // shut off after compressor cycle off  [s]
```

Comment:
Mostly clang-format comment alignment changes. Important changes are removal of data members that are now captured in child coil objects.